src/Platform/Microsoft.Testing.Platform/OutputDevice/IHotReloadPlatformOutputDevice.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.OutputDevice;
+
+internal interface IHotReloadPlatformOutputDevice : IPlatformOutputDevice
```

Comment:
Let's go as-is for now but I really feel like we should move forward with the concept of capability and check if the registered platform output device is "hot reload aware".

Actually looking at the hot reload usage of this interface, I am wondering if it's at all needed? Why don't we simply always call the start/end methods of the `IPlatformOutputDevice` when the hot reload mode is enabled?

Reply:
This is because if you do that you will get Test run aborted message when you press CTRL+C, due to the nature in which hot reload works. This message is confusing because most often no tests will be running at that time. 

These two additional methods are a way to explicitly indicate what is happening, because otherwise we don't know which   abort is the last one.

Reply:
Yes exactly why I think having a capability you could check when in the init/end to display or not the message would be better. As said, this is more of a where we should go than what this PR needs to do.